Client/Frontend/Popup/PopupView.swift

```diff
@@ -85,8 +85,15 @@ class PopupView: UIView, UIGestureRecognizerDelegate {
     var showHandler: (() -> Void)?
     var dismissHandler: (() -> Void)?
     
-    var dialogWidth: CGFloat {
+    var dialogWidth: CGFloat {        
         get {
+            if let superview = self.superview {
+                return min(superview.bounds.width - padding * 2.0, kPopupDialogMaxWidth)
```

Comment:
Lol I meant replace all instances of the "applicationWindow" use when its used for layout. I don't think any of that logic is necessary since you should always have the parents size once added as a subview (including https://github.com/brave/brave-ios/pull/1617/files#diff-fc70fbc4c053eb033f226b1d5320ba43R254)

Reply:
If that set of changes update she scope too much we can fix in a separate ticket/PR